### PR TITLE
Update windows build instructions

### PR DESCRIPTION
**Pull request by dschmidt**
[Rendered](https://github.com/owncloud/client/blob/windows-build-doc/docs/modules/ROOT/pages/building.adoc#windows-development-build-with-kde-craft)

### CODE REVIEW

File: docs/modules/ROOT/pages/building.adoc [hunk 1]
```diff
@@ -107,11 +107,131 @@ Contrary to earlier versions, version 1.7 and later are packaged as a `pkg` inst
 Do not call `make package` at any time when compiling for OS X, as this will build a disk image, which will not work correctly.
 ====
 
-[[windows-development-build]]
-== Windows Development Build
+[[windows-development-build-craft]]
```
Comment (settermjd) [inline docs/modules/ROOT/pages/building.adoc]: You don't need to add a custom anchor, as [they're generated automatically](https://asciidoctor.org/docs/user-manual/#auto-generated-ids).

Comment (dschmidt) [reply docs/modules/ROOT/pages/building.adoc]: I was just following the pattern from the rest of the file :man_shrugging:

**Comment by dschmidt** [comment]
@ckamm yeah, it was about time :-)

File: docs/modules/ROOT/pages/building.adoc (hunk 1 above)
Comment (settermjd) [reply docs/modules/ROOT/pages/building.adoc]: Fair call. It's a new best practice that I'm doing my best to phase in.

**Comment by dschmidt** [comment]
@msetter Totally, as I said I'm not a technical writer - I'm happy if someone takes over with the right skills :)

**Pull request merged**